doc/_templates/index.html

```diff
@@ -111,27 +111,117 @@ <h4 class="card-title">Less data wrangling, more machine learning</h4>
               easily create tabular-learning pipelines that wrangle complex dataframes.
           </p>
       </div>
+      <details>
+	<summary>Given, a complex dataframe
+	<code class="docutils literal notranslate"><span
+class="pre">df</span></code>: <i>(expand for full code)</i></summary>
       <div class="doctest highlight-default notranslate">
         <div class="highlight">
           <pre style="overflow-x: auto;">
 <span class="gp">&gt;&gt;&gt; </span><span class="kn">from</span> <span class="nn">skrub.datasets</span> <span class="kn">import</span> <span class="n">fetch_employee_salaries</span>
 <span class="gp">&gt;&gt;&gt; </span><span class="n">dataset</span> <span class="o">=</span> <span class="n">fetch_employee_salaries</span><span class="p">()</span>
-<span class="gp">&gt;&gt;&gt; </span><span class="n">dataset</span><span class="o">.</span><span class="n">X</span><span class="o">.</span><span class="n">iloc</span><span class="p">[</span><span class="mi">0</span><span class="p">]</span>
-<span class="go">gender                                                                     F</span>
-<span class="go">department                                                               POL</span>
-<span class="go">department_name                                         Department of Police</span>
-<span class="go">division                   MSB Information Mgmt and Tech Division Records...</span>
-<span class="go">assignment_category                                         Fulltime-Regular</span>
-<span class="go">employee_position_title                          Office Services Coordinator</span>
-<span class="go">date_first_hired                                                  09/22/1986</span>
-<span class="go">year_first_hired                                                        1986</span>
-<span class="go">Name: 0, dtype: object</span>
-<span class="go"></span>
+<span class="gp">&gt;&gt;&gt; </span><span class="n">df</span> <span class="o">=</span> <span class="n">dataset</span><span class="o">.</span><span class="n">X</span>
+<span class="gp">&gt;&gt;&gt; </span><span class="n">y</span> <span class="o">=</span> <span class="n">dataset</span><span class="o">.</span><span class="n">y</span>
+<span class="gp">&gt;&gt;&gt; </span><span class="n">df</span></pre>
+        </div>
+      </div>
+    </details>
+<div class="output_subarea output_html rendered_html output_result">
+<div>
```

Comment:
```suggestion
<div style="overflow-x: auto;">
```

(or some other way of making the table scrollable)

Reply:
also why is the details summary black when we hover over it? 
![screenshot_2024-07-02T21:44:57+02:00](https://github.com/skrub-data/skrub/assets/9196501/c24c7b0b-7868-4f8e-99bd-2355e5c99b0e)